pages/home.js

```diff
@@ -65,7 +65,7 @@ export default function Home(props) {
         throw t.invalidNameError
       }
       //Check if name contains special characters
-      else if (!/^[a-zA-Z0-9]+$/.test(username)) {
+      else if (!/^([A-Za-z0-9\s\-\'?])+$/.test(username)) {
```

Comment:
The regex should be pulled from a common spot between both fields that use it

Reply:
I will leave this for now.